lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -453,6 +453,32 @@ export class BrowserCacheManager extends CacheManager {
         });
     }
 
+    /**
+     * Checks the cache for accounts matching loginHint or SID
+     * @param loginHint 
+     * @param sid 
+     */
+    getAccountInfoByHints(loginHint?: string, sid?: string): AccountInfo | null {
```

Comment:
This is useful beyond WAM-JS bridge?

Reply:
What if two accounts have the same email id - translating to same `loginHint` and no `sid`?

Reply:
Added an error for this case